emscripten.py

```diff
@@ -929,6 +931,63 @@ def create_invoke_wrappers(invoke_funcs):
   return invoke_wrappers
 
 
+def create_wasm64_wrappers(metadata):
+  # TODO(sbc): Move this into somewhere less static.  Maybe it can become
+  # part of library.js file, even though this metadata relates specifically
+  # to native (non-JS) functions.
+  #
+  # The signature format here is similar to the one used for JS libraries
+  # but with the following as the only valid char:
+  #  '_' - non-pointer argument (pass though unchanged)
+  #  'p' - pointer/int53 argument (convert to/from BigInt)
+  #  'P' - same as above but allow `undefined` to (required extra check)
+  # but any non-pointer arguments should just be ignore as _, and P i
```

Comment:
I'm not sure what this last sentence is saying?

Reply:
Removed